src/pages/blog/airship-update-june-2020.md

```diff
@@ -0,0 +1,165 @@
+---
+templateKey: blog-post
+title: Airship Update - June 2020
+author: Alexander Hughes
+date: 2020-06-15T12:00:00.000Z
+category: 
+  - label: Airship 2.0
+    id: category-C98iZYrE1
+---
+
+What follows is the Airship Community Update for the month of June 2020, brought to you by the Airship Technical
+Committee.<!-- more -->
+
+## **AIRSHIP BLOG**
+
+The Airship blog is a great way to keep up with what's going on in the community. The Airship community publishes
+[*blog posts*](https://www.airshipit.org/blog/) regularly, including the recent Airship 2.0 Blog series. These blog
+posts introduce the changes from Airship 1.0 to Airship 2.0, highlight new features, and detail the evolution of each
+component. The first six Airship 2.0 Blog posts are already available, and illustrate the design thoughts on the road to
+the alpha milestone:
+
+- [*Airship Blog Series 1 - Evolution Towards 2.0*](
+  https://www.airshipit.org/blog/airship-blog-series-1-evolution-towards-2.0.html)
+- [*Airship Blog Series 2 - An Educated Evolution*](
+   https://www.airshipit.org/blog/airship-blog-series-2-an-educated-evolution.html)
+- [*Airship Blog Series 3 - Airship 2.0 Architecture High Level*](
+   https://www.airshipit.org/blog/airship-blog-series-3-airship-2.0-architecture-high-level.html)
+- [*Airship Blog Series 4 - Shipyard - an Evolution of the Front Door*](
+   https://www.airshipit.org/blog/airship-blog-series-4-shipyard-an-evolution-of-the-front-door.html)
+- [*Airship Blog Series 5 - Drydock and Its Relationship to Cluster API*](
+  https://www.airshipit.org/blog/airship-blog-series-5-drydock-and-its-relationship-to-cluster-api.html)
+- [*Airship Blog Series 6 - Armada Growing Pains*](
+   https://www.airshipit.org/blog/airship-blog-series-6-armada-growing-pains.html)
+
+Stay tuned for more exciting news on the Airship blog series as Airship2.0 progresses through the beta milestone and
+beyond!
+
+<br>
+
+## **JUNE VIRTUAL PTG**
+
+On 04-05 June Airship participated in the OSF hosted Virtual PTG. The PTG facilitated productive cross-team
+collaborations with StarlingX, Ironic and the Edge Working Group. Feed back has been very positive, what follows is a
+summary of the PTG:
+
+In general the attendees were satisfied with the times selected striking a good balance across timezones, between
+virtual and PTG. Most attendees were also in favor of the focused 2 day PTG that did not require full focus for an
+entire week.
+
+Conversations with Starling-X included an update on Airship2.0, and laid out the deprecation cycle for Armada, an
+Airship project being leveraged by Starling-X which is not being developed further in Airship2.0, for more details see
```

Comment:
I found this sentence tough to process. I think it would help if "For more details" started a new sentence.

Reply:
Separated and made more clear.